server/nuxeo-nxr-server/src/main/resources/templates/common-base/client/scripts/taskGC.groovy

```diff
@@ -0,0 +1,33 @@
+import org.apache.logging.log4j.LogManager
+import org.nuxeo.ecm.platform.task.Task
+import org.nuxeo.ecm.core.api.IdRef
+import org.nuxeo.ecm.core.api.ScrollResult
+import org.nuxeo.runtime.transaction.TransactionHelper
+import java.util.HashMap
+
+log = LogManager.getLogger("taskGC.groovy")
+nxql = "SELECT * FROM Document WHERE ecm:mixinType = 'Task'"
+dryRun = true
+session = Context.getCoreSession()
+ScrollResult<String> scrollResult = session.scroll(nxql, 500, 60)
```

Comment:
I'm just wondering, is `500` not too much as batch size to avoid transaction timeout? Or could it be increased? Or `500` is just fine :)
@kevinleturc @bdelbosc ?

Reply:
`500` is the default we put everywhere, it looks like a good default.

I think we always increase it when it comes to production.